config/odp-linux-generic.conf

```diff
@@ -18,6 +18,16 @@
 odp_implementation = "linux-generic"
 config_file_version = "0.0.1"
 
+# Internal shared memory allocator
+ishm: {
+	# ODP will try to reserve as many huge pages as the number indicated
+	# here, up to 64. Zero or a negative value means that no pages should
+	# be reserved.
+	# These pages will only be freed when the application call
+	# odp_term_global().
+	num_reserved_hp = 0
+}
```

Comment:
The comment should state more clearly that this option affects only the number of cached huge pages. More pages can still be reserved, but not cached. A warning about not supporting process mode would also be helpful.

`num_cached_hp` would in my option be more clear name for this variable.

Reply:
Done!